packages/drivers/tinylicious-driver/src/insecureTinyliciousTokenProvider.ts

```diff
@@ -14,14 +17,27 @@ import { v4 as uuid } from "uuid";
  * to get up and running.
  */
 export class InsecureTinyliciousTokenProvider implements ITokenProvider {
-    public async fetchOrdererToken(tenantId: string, documentId?: string): Promise<ITokenResponse> {
+    constructor(
+        /**
+         * Optional. Override of scopes.
```

Comment:
Maybe short description what parameter is and what default plays out if permissions not passed.